src/index.ts

```diff
@@ -0,0 +1,41 @@
+export {
+  BaseAsyncIterReader,
+  AsyncIterReader,
+  LimitReader,
+  NoConcatInflator,
+  StatusAndHeadersParser,
+  StatusAndHeaders,
+  WARCParser,
+  WARCSerializer,
+  WARCRecord,
+  WARC_1_0,
+  WARC_1_1,
+  Indexer,
+  CDXIndexer,
+  postToGetUrl,
+  getSurt,
+  appendRequestQuery,
+  jsonToQueryString,
+  mfdToQueryString,
+  concatChunks,
+} from "./lib";
+
+export type {
+  WARCParserOpts,
+  WARCSerializerOpts,
+  WARCRecordOpts,
+  WARCType,
+} from "./lib";
+
+export { main, indexCommandArgs, cdxIndexCommandArgs } from "./commands";
```

Comment:
This probably shouldn't be exported as part of the main export, as that breaks the browser build in webpack, as it brings in yargs. This should only be part of the cli. The `Indexer` and `CDXIndexer` can be used directly without any node dependencies.